### PR TITLE
Update SC Getting Started Guide

### PR DESCRIPTION
**Pull request by fhildeb**

### CODE REVIEW

File: docs/learn/smart-contract-developers/getting-started.md [hunk 1]
```diff
@@ -4,132 +4,248 @@ sidebar_position: 1
...
+/>
 
-Once finished, you have a working Hardhat setup!
+</div>
 
-## Install packages &amp; setup tools
+## Set up a Hardhat project
 
-To work in the best condition possible, we will install libraries that includes tools, helpers and the [`@lukso/lsp-smart-contracts`](https://www.npmjs.com/package/@lukso/lsp-smart-contracts) package.
+First, create a new folder for the HardHat repository:
```
Comment (Hugoo) [inline docs/learn/smart-contract-developers/getting-started.md:36]: I think Hardhat without H on hat

**Review by Hugoo — APPROVED** [review]
I did half of a review

File: docs/learn/smart-contract-developers/getting-started.md [hunk 2]
```diff
@@ -4,132 +4,342 @@ sidebar_position: 1
...
+      accounts: process.env.PRIVATE_KEY ? [process.env.PRIVATE_KEY] : [],
+    },
+    luksoMainnet: {
+      url: 'https://rpc.lukso.gateway.fm',
+      chainId: 42,
+      accounts: process.env.PRIVATE_KEY ? [process.env.PRIVATE_KEY] : [],
+    },
+  },
+  etherscan: {
+    apiKey: 'no-api-key-needed',
```
Comment (Hugoo) [inline docs/learn/smart-contract-developers/getting-started.md:132]: maybe

```suggestion
    // apiKey: '', - No API key needed for LUKSO explorer
```

File: docs/learn/smart-contract-developers/getting-started.md [hunk 3]
```diff
@@ -4,132 +4,342 @@ sidebar_position: 1
...
+        },
+      },
+    ],
   },
 };
+
+export default config;
+```
+
+## Utilize LSP smart contract presets
```
Comment (Hugoo) [inline docs/learn/smart-contract-developers/getting-started.md:157]: Maybe just "Use"

```suggestion
## Use LSP smart contract presets
```

File: docs/learn/smart-contract-developers/getting-started.md [hunk 4]
```diff
@@ -4,132 +4,342 @@ sidebar_position: 1
...
+
+To utilize LSP smart contracts within your Hardhat project, you can install the latest version of the [`@lukso/lsp-smart-contracts`](https://www.npmjs.com/package/@lukso/lsp-smart-contracts) package like the following:
```
Comment (Hugoo) [inline docs/learn/smart-contract-developers/getting-started.md:159]:
```suggestion
To use LSP smart contracts within your Hardhat project, you can install the latest version of the [`@lukso/lsp-smart-contracts`](https://www.npmjs.com/package/@lukso/lsp-smart-contracts) package like the following:
```

File: docs/learn/smart-contract-developers/getting-started.md [hunk 5]
```diff
@@ -4,132 +4,342 @@ sidebar_position: 1
...
+
+:::success Contract Examples
+
+You can find ready-to-go example contracts within the [`lukso-playground`](https://github.com/lukso-network/lukso-playground) repository.
+
+:::
+
+:::warning
+
+Please remember to always be careful when deploying smart contracts. Do not forget to process a security audit before deploying your smart contracts on production.
```
Comment (Hugoo) [inline docs/learn/smart-contract-developers/getting-started.md:325]: "to process a security audit" sounds weird to me

File: docs/learn/smart-contract-developers/getting-started.md [hunk 6]
```diff
@@ -4,132 +4,342 @@ sidebar_position: 1
...
+
+:::
+
+## Verify smart contracts
+
+To verify a deployed contract, you can use the blockscout API properties set up within the `hardhat.config.ts`. You have to pass a file with all the constructor arguments that you've defined when deploying the smart contract, as well as the address of the deployed smart contract. The parameters and the compiled contract code are then used to verify the payload of the deployed contract on the address.
```
Comment (Hugoo) [inline docs/learn/smart-contract-developers/getting-started.md:331]: maybe you can link:

`blockscout API` to https://explorer.execution.mainnet.lukso.network/api-docs

**Review by Hugoo — COMMENTED** [review]
i didnt run the code to check

File: docs/learn/smart-contract-developers/getting-started.md (hunk 2 above)
Comment (fhildeb) [reply docs/learn/smart-contract-developers/getting-started.md:132]: The entry (`apiKey`) is necessary to run the verify command. If it is not needed, you can put whatever you want

File: docs/learn/smart-contract-developers/getting-started.md (hunk 6 above)
Comment (fhildeb) [reply docs/learn/smart-contract-developers/getting-started.md:331]: That is actually a great find- thanks, will add it (but will put testnet one)

**Comment by fhildeb** [comment]
I checked the tested the code within Hardhat. LGTM

**Pull request merged**